.github/workflows/linter.yml

```
@@ -0,0 +1,25 @@
+name: ESLinter
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    lint-check:
+        name: Lint checker
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            
+            - name: Setup Node.js
+              uses: actions/setup-node@v4
+              with:
+                node-version: 22.14.0
+                cache: 'npm'
+                cache-dependency-path: './package-lock.json'
+
+            - name: clean-install deps
+              uses: npm ci
+
+            - name: lint check
+              uses: eslint src/
```

This should be `run` not `uses` as far as I know
```suggestion
            - name: clean-install deps
              run: npm ci

            - name: lint check
              run: eslint src/
```